cpp/include/cudf/types.hpp

```diff
@@ -32,17 +32,20 @@
  * @brief Type declarations for libcudf.
  */
 
+// FIXME: where is this used? TODO: remove if unused.
 namespace bit_mask {
 using bit_mask_t = uint32_t;
 }
 
```

Comment:
`bit_mask_t` is unused.
Can we remove this?

Reply:
Yes. We have `bitmask_type` defined in this same file and that is what is being used.